### ISSUE TITLE
FlatPak completely problen in latest updates

### ISSUE THREAD

**Issue by ZaxLofful** [flathub/com.parsecgaming.parsec#30]
I have been using the FlatPak install for about 3 weeks now and I was really enjoying it, I had never used the FlatPak version previously; just tried to force it to install. With the latest rounds of OS updates, Parsec fails to launch with the following error:

[Error Screenshot](https://imgur.com/a/tBF2iXy)

It reads: "Parsec could not create the application. Please contact us if this problem persists."

I searched online and asked in the Parsec Dicsord; this error was unknown to any of them. So I am left with its an error of the FlatPak.

It worked for a while, but it stopped after I upgraded; here is a list of the packages that were upgraded:

alsa-ucm-conf
apport
apport-kde
distro-info-data
libldb2
liblouis-data
liblouis20
libnvidia-cfg1-515
libnvidia-common-515
libnvidia-compute-515
libnvidia-compute-515i386:
libnvidia-decode-515
libnvidia-decode-515i386:
libnvidia-encode-515
libnvidia-encode-515i386:
libnvidia-extra-515
libnvidia-fbc1-515
libnvidia-fbc1-515i386:
libnvidia-gl-515
libnvidia-gl-515i386:
libsmbclient
libwbclient0
linux-firmware
linux-modules-nvidia-515-5.19.0-21-generic
linux-modules-nvidia-515-5.19.0-38-generic
linux-modules-nvidia-515-generic
linux-objects-nvidia-515-5.19.0-38-generic
linux-signatures-nvidia-5.19.0-38-generic
nvidia-compute-utils-515
nvidia-driver-515
nvidia-kernel-common-515
nvidia-kernel-source-515
nvidia-utils-515
python3-apport
python3-ldb
python3-problem-report
python3-samba
samba-common
samba-common-bin
samba-dsdb-modules
samba-libs
smbclient
vim-common
vim-tiny
xserver-xorg-video-nvidia-515
xxd

**Comment by ZaxLofful** [comment]
This week showed another round of updates, which now allow Parsec to run just fine....I thought FlatPaks were supposed to be dependency sand-boxing and shouldn't be affected by this....Wonder if it was a bug of FlatPak itself.

Here is a list of the packages that were updated this time (second time):
libldap-2.5-0/kinetic-updates 2.5.14+dfsg-0ubuntu0.22.10.2 amd64 [upgradable from: 2.5.14+dfsg-0ubuntu0.22.10.1]
libldap-2.5-0/kinetic-updates 2.5.14+dfsg-0ubuntu0.22.10.2 i386 [upgradable from: 2.5.14+dfsg-0ubuntu0.22.10.1]
libldap-common/kinetic-updates,kinetic-updates 2.5.14+dfsg-0ubuntu0.22.10.2 all [upgradable from: 2.5.14+dfsg-0ubuntu0.22.10.1]
libllvm15/kinetic-updates 1:15.0.7-0ubuntu0.22.10.1 amd64 [upgradable from: 1:15.0.6-3~ubuntu0.22.10.2]
libllvm15/kinetic-updates 1:15.0.7-0ubuntu0.22.10.1 i386 [upgradable from: 1:15.0.6-3~ubuntu0.22.10.2]
libnss-systemd/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
libpam-systemd/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
libsystemd-shared/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
libsystemd0/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
libsystemd0/kinetic-updates 251.4-1ubuntu7.3 i386 [upgradable from: 251.4-1ubuntu7.1]
libudev1/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
libudev1/kinetic-updates 251.4-1ubuntu7.3 i386 [upgradable from: 251.4-1ubuntu7.1]
systemd-resolved/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
systemd-sysv/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
systemd-timesyncd/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
systemd/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]
udev/kinetic-updates 251.4-1ubuntu7.3 amd64 [upgradable from: 251.4-1ubuntu7.1]

**Comment by SISheogorath** [comment]
It might be. The problems described above seem to originate from GPU drivers and device handling, meaning: kernel-level adjustments. Flatpak only handles the user-space. Even more so, specific parts of the user-space, so while we do dependency sandboxing for the application in the user-space, if your device drivers are incompatible or your kernel simply doesn't understand what the flatpaked dependencies want from it, there will be no happy end.

In short: Whatever the problem was here, it was most likely out of scope of this flatpak.

**Comment by ZaxLofful** [comment]
It's no longer an issue, but it wasn't "unrelated" it required the systems version of flatpak to be updating (because the latest flatpak hadn't made it to the repo yet).

I also run on Kubuntu 23.10 now and it works, so I can no longer vouch for my initial results; that machine has been decommed partly because of this.
